### DISCUSSION TITLE
OSCAL Capabilities Matrix

### DISCUSSION THREAD

**Discussion by xee5ch** [oscal-club/community#24] [Ideas]
I heard an interesting project idea posed in this week's OSCAL Developer Lunch, and I think this one is worth pursuing in the near to medium term. I resurfaced it [in the Gitter OSCAL channel](https://gitter.im/usnistgov-OSCAL/Lobby?at=619698e242fc2863b21594d2):

> So, re today's dev meeting, is there community interest from people like me and oscal.club to start a capabilities listing for OSCAL tools and allow vendors to check boxes and say what is supported with their particular software? I was thinking of a list with basic capabilities needed, but then letting a review process some up where it gets more advanced.
>    Import catalogs in XML
>    Import catalogs in JSON
>    Import catalogs in YAML
>    Resolve profiles in XML
>    Resolve profiles in JSON
>    Resolve profiles in YAML
>    Create SSPs in X/Y/Z format
>    Process OSCAL leveraged authorizations and validate them in X/Y/Z format
>    Cross-referencing SAP/SAR/POA&Ms correctly in X/Y/Z format

I would love feedback on what people envision as a useful set of capabilities for OSCAL friendly tooling. Tell us what capabilities you expect from OSCAL-friendly tooling, conventional GRC application, niche software, whatever! Thanks.

**Comment by rgauss** [comment]
This sounds similar to [CmisCompat](http://cmissync.org/CmisCompat/) for the [CMIS standard](http://docs.oasis-open.org/cmis/CMIS/v1.1/cs01/CMIS-v1.1-cs01.html), could certainly be useful.

I'll give the specific capabilities some thought and post here if they're not already covered by then.

**Reply by xee5ch** [reply]
I love when the community teaches me something new, @rgauss! I was thinking of making an app to compare and contrast in a React Table. Maybe that will supersede [awesome-oscal](https://github.com/oscal-club/awesome-oscal). Not sure yet.

Obviously, will need community feedback. :-)
